docs/releases/1.8.txt

```diff
@@ -457,6 +457,11 @@ Migrations
 * A :ref:`generic mechanism to handle the deprecation of model fields
   <migrations-removing-model-fields>` was added.
 
+* The :class:`~django.db.migrations.operations.RunPython` and
+  :class:`~django.db.migrations.operations.RunSQL` operations now have a
+  ``noop`` class attribute that can be used to mark respective forward or
+  backwards operations as no-ops.
```

Comment:
I'd give an example here: e.g. "This is particularly useful to mark ``RunPython`` or ``RunSQL`` operations as reversible".